cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -285,7 +285,7 @@ func (a *StaticAutoscaler) RunOnce(currentTime time.Time) errors.AutoscalerError
 			scaleDownStart := time.Now()
 			metrics.UpdateLastTime("scaleDown", scaleDownStart)
 			result, typedErr := scaleDown.TryToScaleDown(allNodes, allScheduled, pdbs)
-			metrics.UpdateDuration("scaleDown", scaleDownStart)
```

Comment:
I would consider completely removing 'scaleDown' label. I don't think there is any case where we want to look at node deletion + calculations together.